### DISCUSSION TITLE
Erweiterte CheckIn-Zeiten bei Events wie einreichen/sehen?

### DISCUSSION THREAD

**Discussion by britiger** [Traewelling/traewelling#2923] [Questions / Support] [answered]
Hallo,

ich hab gesehen, dass mit #1668 die CheckIns zu Events vor und nach der eigentlichen Veranstaltung (#1593) umgesetzt wurde. Wie kann man das als Veranstalter entsprechend einreichen bzw. einsehen? Ist das nur im Backend einstellbar und einsehbar? Ich bin im Orga-Team der FOSSGIS-Konferenz und wir haben Aufbau schon am Vortag, Abreise tw. auch am Tag nach dem Event. Also +/- 1 Tag zum Event. Beim CCC ist das ja meinst noch weitreichender. Wäre vielleicht gut wenn man immer einen Puffer für 1 Tag um die Events macht?

**Comment by MrKrisKrisu** [comment, marked as answer]
> Wie kann man das als Veranstalter entsprechend einreichen bzw. einsehen?

Dafür gibt es für User tatsächlich keine Möglichkeit. Wir haben das bisher beim approven der Vorschläge "nach Bauchgefühl" im Backend nachjustiert (oder wenn uns mal jemand darauf explizit angesprochen hat).

> Ich bin im Orga-Team der FOSSGIS-Konferenz und wir haben Aufbau schon am Vortag, Abreise tw. auch am Tag nach dem Event.

Zur FOSSGIS finde ich gerade nur ein Event (ID 379) im März 2024. Soll ich das gerade noch bei einem expliziten Event ändern?

> Wäre vielleicht gut wenn man immer einen Puffer für 1 Tag um die Events macht?

Es wäre glaube generell schöner, wenn man mehr Management der Events in die Hände von Usern legen würde. Dazu müsste das aber jemand entwickeln.

**Reply by britiger** [reply]
Danke für die Info, FOSSGIS 2025 habe ich gerade ins Formular eingetragen. 26-29.3. ist der Zeitraum der Konferenz 25-30.3. (+/- 1 Tag) sollte als CheckIn Zeitraum passen, kannst du dann gern anpassen.
Ich weiß, dass ich dieses Jahr bei der Anfahrt nach gesucht und nichts gefunden habe. Leider kann man das nicht mehr nachtragen.

**Reply by MrKrisKrisu** [reply]
Habe es angepasst.

![image](https://github.com/user-attachments/assets/a4361ec1-00bd-45be-8b67-75500c10d3b8)

**Reply by MrKrisKrisu** [reply]
> Leider kann man das nicht mehr nachtragen.

Zumindest nicht im Frontend. Du kannst über [unsere API die ID des Events](https://traewelling.de/api/documentation#/Status/updateSingleStatus) bei einem Status ändern (letztes Jahr war es wie gesagt 379).
